vsts/vsts.yaml

```diff
@@ -604,13 +597,10 @@ jobs:
           # TODO #181 Samples / tests fail the test due to fake connection strings.
           debugMode: false
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-vulnerabilityassessment.VulnerabilityAssessment@0
-        displayName: "Run Vulnerability Assessment"
-
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@2
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
         displayName: "Publish Security Analysis Logs"
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-report.SdtReport@1
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-report.SdtReport@2
```

Comment:
##[warning]* Please upgrade the version of this build task in your build to major version: 2. For more information, please see: https://aka.ms/gdn-v1-deprecation